src/ucs/vfs/base/vfs_obj.c

```diff
@@ -121,12 +121,14 @@ void ucs_vfs_show_primitive(void *obj, ucs_string_buffer_t *strb, void *arg_ptr,
             ulvalue = *(unsigned long*)arg_ptr;
             break;
         default:
+            ucs_warn("Incorrect type %lu of arg_ptr %p", arg_u64, arg_ptr);
```

Comment:
1. smallcase
2. can we print the vfs path?
`ucs_warn("vfs object %p (%s): incorrect type 0x%lx of arg_ptr %p", obj, path, arg_u64, arg_ptr)`

Reply:
only if we pass additional argument to text callback.
currently it is `(*ucs_vfs_file_show_cb_t)(void *obj, ucs_string_buffer_t *strb, void *arg_ptr, uint64_t arg_u64)`

Reply:
> only if we pass additional argument to text callback.
> currently it is `(*ucs_vfs_file_show_cb_t)(void *obj, ucs_string_buffer_t *strb, void *arg_ptr, uint64_t arg_u64)`

for the path? i guess we could find path by obj ptr, or you wanted to avoid it?

Reply:
We will get path to directory representing the object, not path to file where we get the issue.
Maybe it is better than nothing :)

Reply:
right, maybe reword:
ucs_warn("vfs object %p (%s) attribute %p: incorrect type 0x%lx of arg_ptr %p", obj, path, arg_ptr, arg_u64)

Reply:
ok, so let's print ptr only, no path